### PR TITLE
MAINT: Move to new recipe format

### PR DESCRIPTION
**Pull request by MridulS**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by MridulS** [comment]
@conda-forge-admin, please rerender

**Comment by MridulS** [comment]
@conda-forge-admin, please rerender

**Comment by MridulS** [comment]
@conda-forge-admin, please rerender

File: recipe/recipe.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+context:
+  name: "networkx"
+  version: 3.4.2
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/n/${{ name }}/${{ name }}-${{ version }}.tar.gz
```
Comment (Schefflera-Arboricola) [inline recipe/recipe.yaml:10]:
```suggestion
schema_version: 1

context:
  version: 3.4.2

package:
  name: networkx
  version: ${{ version }}

source:
  url: https://pypi.io/packages/source/n/networkx/networkx-${{ version }}.tar.gz
```

File: recipe/recipe.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        - networkx.algorithms.approximation
+        - networkx.algorithms.tree
+        - networkx.classes
+        - networkx.generators
+        - networkx.drawing
+        - networkx.linalg
+        - networkx.readwrite
+        - networkx.readwrite.json_graph
+        - networkx.tests
+        - networkx.utils
```
Comment (Schefflera-Arboricola) [inline recipe/recipe.yaml:59]:
```suggestion
        - networkx.utils
  - requirements:
      run:
        - pip
    script:
      - pip check
```

**Review by Schefflera-Arboricola — COMMENTED** [review]
How did you generate the new recipe?

I tried generating it locally by referring to [this](https://prefix.dev/blog/rattler_build_on_conda_forge#try-it-out) and following were the differences.

File: recipe/recipe.yaml (hunk 1 above)
Comment (MridulS) [reply recipe/recipe.yaml:10]: Thanks for the review @Schefflera-Arboricola !

`schema_version` is optional and is 1 by default https://github.com/prefix-dev/recipe-format/blob/be58bcc167cb743813a32d258fb70639611474f9/schema.json#L3563

The other changes are just aesthetic changes :)

File: recipe/recipe.yaml (hunk 2 above)
Comment (MridulS) [reply recipe/recipe.yaml:59]: We don't need to do this anymore. `pip check` runs by default https://prefix-dev.github.io/rattler-build/latest/converting_from_conda_build/#converting-the-test-section

File: recipe/recipe.yaml (hunk 1 above)
Comment (MridulS) [reply recipe/recipe.yaml:10]: I used `conda-recipe-manager` too locally, but it is supposed to be a suggestion and we can iterate on the converted output.

**Review by Schefflera-Arboricola — APPROVED** [review]
LGTM! Thanks @MridulS :)

**Comment by MridulS** [comment]
@conda-forge-admin, please rerender

**Comment by bollwyvl** [comment]
It looks like this introduced an unintended side effect:

```
diffoscope --markdown=diff.md info-networkx-3.4.2-pyh*_1* info-networkx-3.4.2-pyh*_2*
```


### info/index.json

#### Pretty-printed

```diff
@@ -1,22 +1,20 @@
 {
-    "arch": null,
-    "build": "pyhd8ed1ab_1",
-    "build_number": 1,
+    "build": "pyh267e887_2",
+    "build_number": 2,
     "constrains": [
+        "numpy >=1.24",
         "scipy >=1.10,!=1.11.0,!=1.11.1",
         "matplotlib >=3.7",
-        "pandas >=2.0",
-        "numpy >=1.24"
+        "pandas >=2.0"
     ],
     "depends": [
-        "python >=3.10"
+        "python >=3.10",
+        "python"
     ],
     "license": "BSD-3-Clause",
-    "license_family": "BSD",
     "name": "networkx",
     "noarch": "python",
-    "platform": null,
     "subdir": "noarch",
-    "timestamp": 1730311574645,
+    "timestamp": 1731521053408,
     "version": "3.4.2"
 }
```

### info/hash_input.json

#### Pretty-printed

```diff
@@ -1,3 +1,4 @@
 {
-    "channel_targets": "conda-forge main"
+    "channel_targets": "conda-forge main",
+    "target_platform": "noarch"
 }
```


This appears to at least be confusing the migrator, which now thinks `networkx` doesn't support py3.13, and will _may_ prevent downstream feedstocks from getting unblocked:

https://conda-forge.org/status/migration/?name=python313

<details>

<summary>full diff...</summary>

# Comparing `info-networkx-3.4.2-pyhd8ed1ab_1.tar.zst` & `info-networkx-3.4.2-pyh267e887_2.tar.zst`

## Comparing `info-networkx-3.4.2-pyhd8ed1ab_1.tar` & `info-networkx-3.4.2-pyh267e887_2.tar`

### file list

```diff
@@ -1,15 +1,11 @@
--rw-r--r--   0        0        0       75 2024-10-30 18:06:14.871524 info/link.json
--rw-r--r--   0        0        0       56 2024-10-30 18:06:14.883525 info/test/run_test.bat
--rw-r--r--   0        0        0        7 2024-10-30 18:06:14.883525 info/test/test_time_dependencies.json
--rw-r--r--   0        0        0        0 2024-10-30 18:06:14.999528 info/git
--rw-r--r--   0        0        0     2038 2024-10-30 18:06:14.883525 info/test/run_test.py
--rw-r--r--   0        0        0      424 2024-10-30 18:06:14.643518 info/index.json
--rw-r--r--   0        0        0    33723 2024-10-30 18:06:14.883525 info/files
--rw-r--r--   0        0        0       43 2024-10-30 18:06:14.639518 info/hash_input.json
--rw-r--r--   0        0        0     1541 2024-10-30 18:04:36.620885 info/recipe/recipe-scripts-license.txt
--rw-r--r--   0        0        0   135632 2024-10-30 18:06:14.999528 info/paths.json
--rw-r--r--   0        0        0       30 2024-10-30 18:06:14.883525 info/test/run_test.sh
--rw-r--r--   0        0        0     6416 2024-10-30 18:06:14.867524 info/about.json
--rw-r--r--   0        0        0      604 2024-10-30 18:06:14.879525 info/recipe/conda_build_config.yaml
--rw-r--r--   0        0        0     2075 2024-10-30 18:03:33.887163 info/recipe/meta.yaml.template
--rw-r--r--   0        0        0     3428 2024-10-30 18:06:14.879525 info/recipe/meta.yaml
+-rw-r--r--   0        0        0      681 2024-11-13 18:04:13.000000 info/about.json
+-rw-r--r--   0        0        0       68 2024-11-13 18:04:13.000000 info/hash_input.json
+-rw-r--r--   0        0        0      375 2024-11-13 18:04:13.000000 info/index.json
+-rw-r--r--   0        0        0     1763 2024-11-13 18:04:13.000000 info/licenses/LICENSE.txt
+-rw-r--r--   0        0        0       75 2024-11-13 18:04:13.000000 info/link.json
+-rw-r--r--   0        0        0   135632 2024-11-13 18:04:13.000000 info/paths.json
+-rw-r--r--   0        0        0     1541 2024-11-13 18:04:13.000000 info/recipe/recipe-scripts-license.txt
+-rw-r--r--   0        0        0     2737 2024-11-13 18:04:13.000000 info/recipe/recipe.yaml
+-rw-r--r--   0        0        0    20526 2024-11-13 18:04:13.000000 info/recipe/rendered_recipe.yaml
+-rw-r--r--   0        0        0       58 2024-11-13 18:04:13.000000 info/recipe/variant_config.yaml
+-rw-r--r--   0        0        0     1446 2024-11-13 18:04:13.000000 info/tests/tests.yaml
```

### filetype from file(1)

```diff
@@ -1 +1 @@
-POSIX tar archive
+POSIX tar archive (GNU)
```

### info/index.json

#### Pretty-printed

```diff
@@ -1,22 +1,20 @@
 {
-    "arch": null,
-    "build": "pyhd8ed1ab_1",
-    "build_number": 1,
+    "build": "pyh267e887_2",
+    "build_number": 2,
     "constrains": [
+        "numpy >=1.24",
         "scipy >=1.10,!=1.11.0,!=1.11.1",
         "matplotlib >=3.7",
-        "pandas >=2.0",
-        "numpy >=1.24"
+        "pandas >=2.0"
     ],
     "depends": [
-        "python >=3.10"
+        "python >=3.10",
+        "python"
     ],
     "license": "BSD-3-Clause",
-    "license_family": "BSD",
     "name": "networkx",
     "noarch": "python",
-    "platform": null,
     "subdir": "noarch",
-    "timestamp": 1730311574645,
+    "timestamp": 1731521053408,
     "version": "3.4.2"
 }
```

### info/hash_input.json

#### Pretty-printed

```diff
@@ -1,3 +1,4 @@
 {
-    "channel_targets": "conda-forge main"
+    "channel_targets": "conda-forge main",
+    "target_platform": "noarch"
 }
```

### info/paths.json

#### Pretty-printed

```diff
@@ -1,64 +1,10 @@
 {
     "paths": [
         {
-            "_path": "site-packages/networkx-3.4.2.dist-info/INSTALLER",
-            "path_type": "hardlink",
-            "sha256": "d0edee15f91b406f3f99726e44eb990be6e34fd0345b52b910c568e0eef6a2a8",
-            "size_in_bytes": 5
-        },
-        {
-            "_path": "site-packages/networkx-3.4.2.dist-info/LICENSE.txt",
-            "path_type": "hardlink",
-            "sha256": "5b433b90f755eb9bbd06feff1d1a4f5f232c5208a185694199e45fa95d762792",
-            "size_in_bytes": 1763
-        },
-        {
-            "_path": "site-packages/networkx-3.4.2.dist-info/METADATA",
-            "path_type": "hardlink",
-            "sha256": "2e5265dda876ef3c44fefa9e9553499bc564c89a16c95e38bdaf9babc556d1d7",
-            "size_in_bytes": 6349
-        },
-        {
-            "_path": "site-packages/networkx-3.4.2.dist-info/RECORD",
-            "path_type": "hardlink",
-            "sha256": "940a647504da9ede6a7bb1e267380882c35e1242aab614b9cd2c2922e487b620",
-            "size_in_bytes": 99452
-        },
-        {
-            "_path": "site-packages/networkx-3.4.2.dist-info/REQUESTED",
-            "path_type": "hardlink",
-            "sha256": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
-            "size_in_bytes": 0
-        },
-        {
-            "_path": "site-packages/networkx-3.4.2.dist-info/WHEEL",
-            "path_type": "hardlink",
-            "sha256": "195f5a3138703ffe28342b6f102d9e737a9462eb6059e033925ae8ff49b85894",
-            "size_in_bytes": 91
-        },
-        {
-            "_path": "site-packages/networkx-3.4.2.dist-info/direct_url.json",
-            "path_type": "hardlink",
-            "sha256": "9c397a0743546bca4d74b02c4c9a7c8a73cffdb43285118d3b831d16bb50649c",
-            "size_in_bytes": 104
-        },
-        {
-            "_path": "site-packages/networkx-3.4.2.dist-info/entry_points.txt",
-            "path_type": "hardlink",
-            "sha256": "1f68d9683b0327f8bd1f64b05a9c2e15e97c06b67dc472aebe1f83400596f654",
-            "size_in_bytes": 94
-        },
-        {
-            "_path": "site-packages/networkx-3.4.2.dist-info/top_level.txt",
-            "path_type": "hardlink",
-            "sha256": "b37324fbb28e96efa40f7f70f1783f297a0fe59fcc56f801faea64cae384e079",
-            "size_in_bytes": 9
-        },
-        {
             "_path": "site-packages/networkx/__init__.py",
             "path_type": "hardlink",
             "sha256": "bd5f9b6329a5f492b9395f0873f75cb4be59191e4dab01787f37789ac4766fd5",
             "size_in_bytes": 1274
         },
         {
             "_path": "site-packages/networkx/algorithms/__init__.py",
@@ -3505,11 +3451,65 @@
             "size_in_bytes": 1579
         },
         {
             "_path": "site-packages/networkx/utils/union_find.py",
             "path_type": "hardlink",
             "sha256": "3712a5065c92ef50355a59eddbc2fec3266823d13166f26d87fd1ed97495ae2b",
             "size_in_bytes": 3338
+        },
+        {
+            "_path": "site-packages/networkx-3.4.2.dist-info/INSTALLER",
+            "path_type": "hardlink",
+            "sha256": "bc33022edcb7639ff53355b4e91dade50a0bbf0299efeb6171d1ec0ba5029cfc",
+            "size_in_bytes": 6
+        },
+        {
+            "_path": "site-packages/networkx-3.4.2.dist-info/LICENSE.txt",
+            "path_type": "hardlink",
+            "sha256": "5b433b90f755eb9bbd06feff1d1a4f5f232c5208a185694199e45fa95d762792",
+            "size_in_bytes": 1763
+        },
+        {
+            "_path": "site-packages/networkx-3.4.2.dist-info/METADATA",
+            "path_type": "hardlink",
+            "sha256": "c68e17726a1967702630cd230fa251d6232a3ed4553e517b06e90c9a9b718812",
+            "size_in_bytes": 6293
+        },
+        {
+            "_path": "site-packages/networkx-3.4.2.dist-info/RECORD",
+            "path_type": "hardlink",
+            "sha256": "b9e9b074bfb71d09b9d89af0f2bcae378240b79a60726f55c39975797c1c7bee",
+            "size_in_bytes": 99452
+        },
+        {
+            "_path": "site-packages/networkx-3.4.2.dist-info/REQUESTED",
+            "path_type": "hardlink",
+            "sha256": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+            "size_in_bytes": 0
+        },
+        {
+            "_path": "site-packages/networkx-3.4.2.dist-info/WHEEL",
+            "path_type": "hardlink",
+            "sha256": "3fd8f0fa01237bc07207bfe15e82029c7b550ab13031087eeb7d2d2af4167a17",
+            "size_in_bytes": 91
+        },
+        {
+            "_path": "site-packages/networkx-3.4.2.dist-info/direct_url.json",
+            "path_type": "hardlink",
+            "sha256": "7c32fdbc63225a038180052dabe3492029621276a610afc5c02d3675da37fd07",
+            "size_in_bytes": 119
+        },
+        {
+            "_path": "site-packages/networkx-3.4.2.dist-info/entry_points.txt",
+            "path_type": "hardlink",
+            "sha256": "1f68d9683b0327f8bd1f64b05a9c2e15e97c06b67dc472aebe1f83400596f654",
+            "size_in_bytes": 94
+        },
+        {
+            "_path": "site-packages/networkx-3.4.2.dist-info/top_level.txt",
+            "path_type": "hardlink",
+            "sha256": "b37324fbb28e96efa40f7f70f1783f297a0fe59fcc56f801faea64cae384e079",
+            "size_in_bytes": 9
         }
     ],
     "paths_version": 1
 }
```

### info/about.json

#### Pretty-printed

```diff
@@ -1,183 +1,16 @@
 {
     "channels": [
-        "https://conda.anaconda.org/conda-forge"
+        "https://conda.anaconda.org/conda-forge/"
     ],
-    "conda_build_version": "24.9.0",
-    "conda_version": "24.9.2",
-    "description": "NetworkX is a Python language software package for the creation,\nmanipulation, and study of the structure, dynamics, and functions of complex\nnetworks.\n",
+    "description": "NetworkX is a Python language software package for the creation,\nmanipulation, and study of the structure, dynamics, and functions of complex\nnetworks.",
     "dev_url": "https://github.com/networkx/networkx",
     "doc_url": "https://networkx.org/documentation/stable/",
-    "env_vars": {
-        "CIO_TEST": "<not set>"
-    },
     "extra": {
-        "copy_test_source_files": true,
-        "final": true,
-        "flow_run_id": "azure_20241030.2.1",
-        "recipe-maintainers": [
-            "stefanv",
-            "synapticarbors",
-            "ocefpaf",
-            "SylvainCorlay",
-            "FelixMoelder",
-            "MridulS"
-        ],
+        "flow_run_id": "azure_20241113.3.1",
         "remote_url": "https://github.com/conda-forge/networkx-feedstock",
-        "sha": "4cc836b2fd94cdd8abf0f0f0ce359a83431f802e"
+        "sha": "2e3e3c327dd4e8ba56afff203d82a09c96396d98"
     },
-    "home": "https://networkx.github.io/",
-    "identifiers": [],
-    "keywords": [],
+    "home": "https://networkx.org/",
     "license": "BSD-3-Clause",
-    "license_family": "BSD",
-    "license_file": "LICENSE.txt",
-    "root_pkgs": [
-        "conda-libmamba-solver 24.9.0 pyhd8ed1ab_0",
-        "libgcc 14.2.0 h77fa898_1",
-        "krb5 1.21.3 h659f571_0",
-        "_libgcc_mutex 0.1 conda_forge",
-        "cffi 1.17.1 py312h06ac9bb_0",
-        "liblief 0.14.1 h5888daf_2",
-        "nbformat 5.10.4 pyhd8ed1ab_0",
-        "pycosat 0.6.6 py312h98912ed_0",
-        "requests-toolbelt 1.0.0 pyhd8ed1ab_0",
-        "py-lief 0.14.1 py312h2ec8cdc_2",
-        "certifi 2024.8.30 pyhd8ed1ab_0",
-        "python-libarchive-c 5.1 py312h7900ff3_1",
-        "lz4-c 1.9.4 hcb278e6_0",
-        "libgomp 14.2.0 h77fa898_1",
-        "six 1.16.0 pyh6c4a22f_0",
-        "su-exec 0.2 h166bdaf_1003",
-        "anaconda-client 1.12.3 pyhd8ed1ab_1",
-        "c-ares 1.32.3 h4bc722e_0",
-        "libmamba 1.5.10 hf72d635_1",
-        "traitlets 5.14.3 pyhd8ed1ab_0",
-        "zstd 1.5.6 ha6fb4c9_0",
-        "yaml-cpp 0.8.0 h59595ed_0",
-        "python-fastjsonschema 2.20.0 pyhd8ed1ab_0",
-        "ruamel.yaml 0.18.6 py312h66e93f0_1",
-        "libexpat 2.6.3 h5888daf_0",
-        "patchelf 0.17.2 h58526e2_0",
-        "idna 3.10 pyhd8ed1ab_0",
-        "conda-build 24.9.0 py312h7900ff3_0",
-        "importlib_resources 6.4.5 pyhd8ed1ab_0",
-        "zipp 3.20.2 pyhd8ed1ab_0",
-        "yaml 0.2.5 h7f98852_2",
-        "distro 1.9.0 pyhd8ed1ab_0",
-        "conda-package-handling 2.4.0 pyh7900ff3_0",
-        "defusedxml 0.7.1 pyhd8ed1ab_0",
-        "more-itertools 10.5.0 pyhd8ed1ab_0",
-        "tzdata 2024b hc8b5060_0",
-        "python 3.12.7 hc5c86c4_0_cpython",
-        "pysocks 1.7.1 pyha2e5f31_6",
-        "bzip2 1.0.8 h4bc722e_7",
-        "packaging 24.1 pyhd8ed1ab_0",
-        "libev 4.33 hd590300_2",
-        "requests 2.32.3 pyhd8ed1ab_0",
-        "reproc 14.2.4.post0 hd590300_1",
-        "charset-normalizer 3.4.0 pyhd8ed1ab_0",
-        "menuinst 2.1.2 py312h7900ff3_1",
-        "ca-certificates 2024.8.30 hbcca054_0",
-        "libstdcxx-ng 14.2.0 h4852527_1",
-        "pluggy 1.5.0 pyhd8ed1ab_0",
-        "libxcrypt 4.4.36 hd590300_1",
-        "jsonpatch 1.33 pyhd8ed1ab_0",
-        "pkginfo 1.11.2 pyhd8ed1ab_0",
-        "hyperframe 6.0.1 pyhd8ed1ab_0",
-        "conda-index 0.5.0 pyhd8ed1ab_0",
-        "python_abi 3.12 5_cp312",
-        "libcurl 8.10.1 hbbe4b11_0",
-        "chardet 5.2.0 py312h7900ff3_2",
-        "libffi 3.4.2 h7f98852_5",
-        "ruamel.yaml.clib 0.2.8 py312h66e93f0_1",
-        "jsonschema 4.23.0 pyhd8ed1ab_0",
-        "libstdcxx 14.2.0 hc0a3c3a_1",
-        "filelock 3.16.1 pyhd8ed1ab_0",
-        "libmambapy 1.5.10 py312hf3f0a4e_1",
-        "xz 5.2.6 h166bdaf_0",
-        "click 8.1.7 unix_pyh707e725_0",
-        "truststore 0.10.0 pyhd8ed1ab_0",
-        "tini 0.19.0 h166bdaf_1",
-        "libiconv 1.17 hd590300_2",
-        "wheel 0.44.0 pyhd8ed1ab_0",
-        "rpds-py 0.20.0 py312h12e396e_1",
-        "psutil 6.1.0 py312h66e93f0_0",
-        "urllib3 2.2.3 pyhd8ed1ab_0",
-        "keyutils 1.6.1 h166bdaf_0",
-        "libnghttp2 1.64.0 h161d5f1_0",
-        "libnsl 2.0.1 hd590300_0",
-        "attrs 24.2.0 pyh71513ae_0",
-        "jinja2 3.1.4 pyhd8ed1ab_0",
-        "libxml2 2.12.7 he7c6b58_4",
-        "conda 24.9.2 py312h7900ff3_0",
-        "colorama 0.4.6 pyhd8ed1ab_0",
-        "beautifulsoup4 4.12.3 pyha770c72_0",
-        "libarchive 3.7.4 hfca40fe_0",
-        "platformdirs 4.3.6 pyhd8ed1ab_0",
-        "hpack 4.0.0 pyh9f0ad1d_0",
-        "pytz 2024.2 pyhd8ed1ab_0",
-        "pip 24.2 pyh8b19718_1",
-        "setuptools 75.1.0 pyhd8ed1ab_0",
-        "openssl 3.3.2 hb9d3cd8_0",
-        "lzo 2.10 hd590300_1001",
-        "libssh2 1.11.0 h0841786_0",
-        "_openmp_mutex 4.5 2_gnu",
-        "jsonschema-specifications 2024.10.1 pyhd8ed1ab_0",
-        "archspec 0.2.3 pyhd8ed1ab_0",
-        "jsonpointer 3.0.0 py312h7900ff3_1",
-        "ld_impl_linux-64 2.43 h712a8e2_2",
-        "pcre2 10.44 hba22ea6_2",
-        "libsqlite 3.47.0 hadc24fc_0",
-        "patch 2.7.6 h7f98852_1002",
-        "git 2.47.0 pl5321h59d505e_0",
-        "brotli-python 1.1.0 py312h2ec8cdc_2",
-        "frozendict 2.4.6 py312h66e93f0_0",
-        "mamba 1.5.10 py312h9460a1c_1",
-        "jupyter_core 5.7.2 pyh31011fe_1",
-        "libgcc-ng 14.2.0 h69a702a_1",
-        "libzlib 1.3.1 hb9d3cd8_2",
-        "boltons 24.0.0 pyhd8ed1ab_0",
-        "h2 4.1.0 pyhd8ed1ab_0",
-        "libuuid 2.38.1 h0b41bf4_0",
-        "pybind11-abi 4 hd8ed1ab_3",
-        "zstandard 0.23.0 py312hef9b889_1",
-        "soupsieve 2.5 pyhd8ed1ab_1",
-        "perl 5.32.1 7_hd590300_perl5",
-        "icu 75.1 he02047a_0",
-        "tqdm 4.66.5 pyhd8ed1ab_0",
-        "libsolv 0.7.30 h3509ff9_0",
-        "conda-package-streaming 0.11.0 pyhd8ed1ab_0",
-        "pkgutil-resolve-name 1.3.10 pyhd8ed1ab_1",
-        "fmt 11.0.2 h434a139_0",
-        "pyyaml 6.0.2 py312h66e93f0_1",
-        "tk 8.6.13 noxft_h4845f30_101",
-        "referencing 0.35.1 pyhd8ed1ab_0",
-        "pycparser 2.22 pyhd8ed1ab_0",
-        "reproc-cpp 14.2.4.post0 h59595ed_1",
-        "markupsafe 3.0.2 py312h178313f_0",
-        "libedit 3.1.20191231 he28a2e2_2",
-        "readline 8.2 h8228510_1",
-        "python-dateutil 2.9.0 pyhd8ed1ab_0",
-        "ripgrep 14.1.1 h8fae777_0",
-        "ncurses 6.5 he02047a_1",
-        "rattler-build-conda-compat 1.2.2 pyhd8ed1ab_0",
-        "conda-forge-ci-setup 4.12.0 py312hb3d6910_100",
-        "oras-py 0.1.14 pyhd8ed1ab_0",
-        "wrapt 1.16.0 py312h66e93f0_1",
-        "joblib 1.4.2 pyhd8ed1ab_0",
-        "jq 1.7.1 hd590300_0",
-        "tomli 2.0.2 pyhd8ed1ab_0",
-        "conda-forge-metadata 0.11.0 pyhd8ed1ab_0",
-        "shyaml 0.6.2 pyhd3deb0d_0",
-        "deprecated 1.2.14 pyh1a96a4e_0",
-        "conda-oci-mirror 0.1.0 pyhd8ed1ab_0",
-        "ruamel_yaml 0.15.80 py312h98912ed_1009",
-        "typing_extensions 4.12.2 pyha770c72_0",
-        "oniguruma 6.9.9 hd590300_0",
-        "rattler-build 0.28.2 h51b9b6e_0",
-        "conda-env 2.6.0 1",
-        "typing-extensions 4.12.2 hd8ed1ab_0"
-    ],
-    "summary": "Python package for creating and manipulating complex networks",
-    "tags": []
+    "summary": "Python package for creating and manipulating complex networks"
 }
```



</details>

**Comment by bollwyvl** [comment]
Raised on [z :tulip:](https://conda-forge.zulipchat.com/#narrow/channel/457337-general/topic/recipe.20format.20v1.20vs.20py313.20migrator)

**Pull request merged**